### DISCUSSION TITLE
Seeking advice regarding breaking changes between 1.9.1 and 1.49.x

### DISCUSSION THREAD

**Discussion by ClosetGeek-Git** [libuv/libuv#4643] [General]
I'm new to the libuv world. I'm thinking about picking up an 8 yr old project that depended on libuv 1.9.1 and I'm curious about how much of a jump this is with libuv. Can anyone give me any input as far as breaking changes that have happened in this period if any? I would love to be able to just build it and have things work, but it's going to take a fair amount of updating just to get things to the point that I can tell if a newer version of libuv will simply "drop in". With my lack of familiarity with libuv I need to be aware of how much I might be  committing to if this too has to be updated within the project, Thank you

**Comment by ClosetGeek-Git** [comment]
The project I have in mind is https://github.com/GPBeta/nc.js . I already know it's going to be a bit of an undertaking

**Comment by saghul** [comment]
In principle, and barring unnoticed bugs, it should be a drop-in replacement. We have script backwards-compatibility in the API / ABI.
